index.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string.
```

Comment:
This would be slightly more precise:

```diff
-   * Retrieves a given environment variable as a string.
+   * Returns the value of the given environment variable as a string.
```